tests/unit/containers/offer-page/faq-block/FaqBlock.spec.jsx

```diff
@@ -0,0 +1,73 @@
+import { beforeEach, expect, vi } from 'vitest'
```

Comment:
We don't have to import these variables because they are set globally.
You can check Vitest configuration and also [documentation](https://vitest.dev/config/#globals).